sites/fast-website/src/public/index.html

```diff
@@ -6,127 +6,114 @@
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
-    <fast-design-system-provider id="body-provider" class="body-provider"
+    <fast-design-system-provider
+      class="body-provider"
       use-defaults
-      background-color="#1F1F1F"
-      accent-base-color="#F33378"
+      background-color="#181818"
```

Comment:
Not sure how stale this target branch is, but the design-system-provider now has both the background and accent color as default values - these can be removed if you're up to date.

Reply:
Just checked, I'll remove them.